internal/utils.ts

```diff
@@ -102,15 +102,14 @@ const preflight = computed((): PreflightResult => {
   if (!host || !isValidHost(endpoint.value))
     return 'host';
 
-  const {
-    location: { hostname },
-  } = window;
+  const hostname = window.location.hostname;
+  const storage = window.localStorage;
```

Comment:
I'm honestly not sure what the difference in access between `window.localStorage` and just using `localStorage` is, but I also can't find any information on the internet about this, so if this also works, I don't have an oponion :D